README.md

```diff
@@ -41,34 +38,100 @@ require('apostrophe')({
   }
 });
 ```
+## Exporting Files
+
+### Pages
+![Screenshot highlighting the export menu item in the context menu of a page in the manager.](images/page-export.png)
+
+Pages can currently only be exported singly. When exporting a page it will retain its publication status upon import. A page that is published when exported will be published on import, while draft pages will remain in draft status. In order to export a page, you use the context menu to the right of each page in the manager.
+
+Exporting the parent page in a set of nested documents will **not** export the child pages. Each child page must be exported separately. The parent page must already exist, or be separately exported and then subsequently imported, for any child page to appear in the page manager of the import project.
+
+<hr>
+
+![Screenshot of the page export modal](images/page-export-modal.png)
+
+Clicking export will bring up a modal with two input controls.
+
+The first is for selecting the file format to use for the exported file. Currently, the module only exports in the `.tar.gz` format, but this will possibly be expanded in the future.
+
+The second input toggles whether documents associated with the page, like images, files, pieces, or relationships, should be included in the download. Any or all can be unselected.Note that this doesn't mean that the particular page includes all of those documents types, just that those document types both exist in your project and aren't disabled for export.
+
+Clicking on the "Export Page" button will trigger a download of the export file to your local computer.
+
+<hr>
+
+### Pieces
+![Screenshot of exporting pieces using the batch method](images/piece-batch-export.png)
+
+Unlike pages, piece-types can be exported either individually from the context menu to the right of the piece, or as a batch using the context menu above the pieces list in the manager. For example, in the image above two articles have been selected for export.
+
+Clicking export will bring up the same modal that appears when exporting pages, allowing for the selection of related documents that should also be exported.
```

Comment:
In user-facing documentation I suggest "dialog box"

Reply:
Changed in multiple locations.